cookiecutter/wrapper-template/cookiecutter.json

```diff
@@ -0,0 +1,9 @@
+{
+  "source_name": "MyExtensionName",
+  "admin_name": "FirstName LastName",
+  "extension_name": "{{ cookiecutter.source_name.lower() }}",
```

Comment:
"Source name" is from 'taps' context, I think, and feels off here. But I see that "extension name" is already taken.

How about any of these:

- `extension_title` (upper) > `extension_name` (lower) 
- `extension_name` (upper) > `library_name` (lower)
- `extension_name` (upper) > `extension_name_lower` (lower)
- `tool_name` (upper) > `extension_name` (lower)

Or something completely different? 🤷

Reply:
Ah I read "source name" as "tell me the name of this source code name" or something , but it originating from "what is this taps source" makes much more sense 😆 

I'll roll with `extension_name (upper) > library_name (lower)`